spec-main/api-web-frame-main-spec.ts

```diff
@@ -143,6 +143,11 @@ describe('webFrameMain module', () => {
 
       expect(webFrame.visibilityState).to.equal('visible');
       w.hide();
+
```

Comment:
Instead of an arbitrary timeout, you could do something like the following:
```
  async function waitForVisibilityState (webFrame, expectedState) {
    let state = webFrame.visibilityState;
    while (webFrame.visibilityState !== expectedState) {
      await new Promise(resolve => setTimeout(resolve, 10));
      state = webFrame.visibilityState;
    }
    return state;
  }
  expect(waitForVisibilityState(webFrame, 'hidden')).to.eventually.equal('hidden');
```

Reply:
I've added a helper for this kind of behavior 9c5ae84